.pre-commit-config.yaml

```diff
@@ -6,21 +6,21 @@ repos:
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
--   repo: https://github.com/psf/black
-    rev: 22.8.0
-    hooks:
-    -   id: black
--   repo: http://github.com/PyCQA/isort
-    rev: 5.10.1
-    hooks:
-    -   id: isort
--   repo: http://github.com/PyCQA/flake8
-    rev: 5.0.4
-    hooks:
-    -   id: flake8
-        exclude: test.py
 -   repo: http://github.com/doublify/pre-commit-rust
     rev: v1.0
     hooks:
-    -   id: fmt
     -   id: cargo-check
```

Comment:
do we need this?

Reply:
Well I can remove it, but we'd need a `make` target for `cargo check`